xls/passes/rematerialization_pass.cc

```diff
@@ -279,7 +367,7 @@ FindRematerializationOpportunitiesAtNode(
       // If `chunk` were empty, that would mean that `child` is available, but
       // we just checked for this in the surrounding `if` statement.
       XLS_CHECK(!chunk.empty());
-      if (std::any_of(chunk.begin(), chunk.end(),
+      if (!std::any_of(chunk.begin(), chunk.end(),
```

Comment:
This change seems closer to correct but I honestly can't figure out what my intention was with that code originally. I wonder if making that change in my version internally would result in quality improvements on our internal benchmarks, since as it is this will result in 0 opportunities being found.